Si702x.class.nut

```diff
@@ -13,61 +16,96 @@ class Si702x {
     static TIMEOUT      = 200; // ms
```

Comment:
Yes, that seems wise. Thanks for the suggestion - please feel free to comment on these things in the future!